remoteworkitem/scheduler.go

```diff
@@ -76,9 +77,9 @@ func (s *Scheduler) ScheduleAllQueries(ctx context.Context, accessTokens map[str
 	cr.Start()
 }
 
-func fetchTrackerQueries(db *gorm.DB) []trackerSchedule {
-	tsList := []trackerSchedule{}
-	err := db.Table("tracker_queries").Select("trackers.id as tracker_id, trackers.url, trackers.type as tracker_type, tracker_queries.query, tracker_queries.schedule, tracker_queries.space_id").Joins("left join trackers on tracker_queries.tracker_id = trackers.id").Where("trackers.deleted_at is NULL AND tracker_queries.deleted_at is NULL").Scan(&tsList).Error
+func fetchTrackerQueries(db *gorm.DB) []TrackerSchedule {
+	tsList := []TrackerSchedule{}
+	err := db.Table("tracker_queries").Select("trackers.id as tracker_id, trackers.url, trackers.type as tracker_type, tracker_queries.query, tracker_queries.schedule, tracker_queries.space_id, tracker_queries.work_item_type_id as work_item_type_id").Joins("left join trackers on tracker_queries.tracker_id = trackers.id").Where("trackers.deleted_at is NULL AND tracker_queries.deleted_at is NULL").Scan(&tsList).Error
```

Comment:
You probably can remove these `... as ...` if you decorate the struct definition with proper tags.

Reply:
please checkout this 72bc5c5

Reply:
Sorry it was my misunderstanding and the missing knowledge that `TrackerSchedule` isn't really a thing in the DB.